Formula/crystal-lang.rb

```diff
@@ -1,15 +1,15 @@
 class CrystalLang < Formula
   desc "Fast and statically typed, compiled language with Ruby-like syntax"
   homepage "https://crystal-lang.org/"
-  revision 3
+  revision 2
```

Comment:
Why is this 2? If it's a new version, the revision line should be removed, not decremented by 1.

Reply:
My bad - I thought it was matching the other platforms build numbers, but it was just a coincidence. I've just dropped this 👍